infra/app/service/main.tf

```diff
@@ -66,11 +69,6 @@ data "aws_rds_cluster" "db_cluster" {
   cluster_identifier = local.database_config.cluster_name
 }
 
-data "aws_iam_policy" "db_access_policy" {
-  count = module.app_config.has_database ? 1 : 0
-  name  = local.database_config.access_policy_name
-}
-
```

Comment:
we need to rename this data source to app_db_access_policy and add a new one migrator_db_access_policy, then pass those in below to the service module